core-web/libs/sdk/angular/src/lib/layout/column/column.component.css

```diff
@@ -0,0 +1,99 @@
+:host.col-start-1 {
```

Comment:
Do we need `:host`?

Reply:
Yes, the class is added to host by `HostBinding`